great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -3841,3 +3848,16 @@ def save_config_variable(
 
         with open(config_variables_filepath, "w") as config_variables_file:
             yaml.dump(config_variables, config_variables_file)
+
+    def _load_zep_config(self) -> GxConfig:
```

Comment:
```suggestion
    def _initialize_zep_config(self) -> GxConfig:
```
This might be a better name

Reply:
I am fine either way and we can change it as his function involves.